spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/util/TaskSanitizer.java

```diff
@@ -42,13 +42,15 @@ public TaskExecution sanitizeTaskExecutionArguments(TaskExecution taskExecution)
 		return taskExecution;
 	}
 
-	public TaskManifest sanitizeTaskManifest(TaskManifest taskManifest) {
+	public TaskExecutionManifest sanitizeTaskManifest(TaskExecutionManifest taskManifest) {
```

Comment:
We can rename this method as well (to have TaskExecutionManifest)

Reply:
Is there a reason the methods in this class are not static?

Reply:
Done (although still wondering why these methods are not static)

Reply:
I don't see any reason why this can't be static. I think we are ok both ways.

Reply:
Well, it would be nice to not have instances of this class floating around everywhere it's needed if we can make them static.  If you're ok with it, I'll make the change.

Reply:
I agree. Though it is only used by the task configuration beans, it makes sense to have a static method for this. Please go ahead!

Reply:
Done